README.md

```diff
@@ -12,7 +12,7 @@
   <a href="https://github.com/forem/forem/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev%22">
     <img src="https://img.shields.io/github/issues/forem/forem/ready for dev" alt="GitHub issues ready for dev">
   </a>
-  <a href="https://gitpod.io/from-referrer/">
+  <a href="https://gitpod.io/#https://github.com/forem/forem">
```

Comment:
This is the correct URL for launching Gitpod, but only if you are a core contributor or want a version of Forem to look at and run.

I'm wondering if providing the format of the URL would be more helpful to potential contributors, e.g.

```markdown
To launch Forem in Gitpod, navigate to https://gitpod.io/#https://github.com/your_github_username/forem
```

Thoughts @amywtlin ?

Reply:
Sorry, I meant the new URL you added is correct, https://gitpod.io/#https://github.com/forem/forem, but only for core contributors.

Reply:
I've got only https://github.com in referral headers. :c
![image](https://user-images.githubusercontent.com/14293805/157125420-f34eee8c-b32a-4405-b160-9dbc824b02ba.png)

Reply:
One way that might help is to stress `your_github_username` with brackets surrounding it so it is more obvious that whoever copying & pasting the URL needs to replace that parameter with their own username for it to work. 

From: 
```To launch Forem in Gitpod, navigate to https://gitpod.io/#https://github.com/your_github_username/forem```
To: 
```To launch Forem in Gitpod, navigate to https://gitpod.io/#https://github.com/{your_github_username}/forem```

I am not sure if we have used one specific type of brackets in this case... if we have, we should stick to the same type. cc @aitchiss as I feel like you would know if we have opted for a certain type of brackets implying it's a parameter.

Reply:
Yeah I would suggest the same as you @amywtlin, with the `{}` brackets 👍